tests/garage/torch/q_functions/test_continuous_mlp_q_function.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from unittest.mock import MagicMock
+
+import akro
+import numpy as np
+import torch
+from torch import nn
+
+from garage.torch.modules import MLPModule
+from garage.torch.q_functions import ContinuousMLPQFunction
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class TestContinuousMLPQFunction(unittest.TestCase):
```

Comment:
please add more comprehensive test cases which test many parameter variations and input/output pairs.

you can see https://github.com/rlworkgroup/garage/blob/master/tests/garage/tf/q_functions/test_continuous_mlp_q_function_with_model.py and https://github.com/rlworkgroup/garage/pull/722 for samples